src/shared/src/types/user_profile.rs

```diff
@@ -8,7 +8,13 @@ use serde::Serialize;
 use super::{verifiable_credential::CredentialType, Timestamp};
```

Comment:
@AntonioVentilii Would you be so kind as to check the length bounds proposed below?